mkdocs.yml

```diff
@@ -179,12 +180,6 @@ nav:
           - Task Runners: concepts/task-runners.md
           - Events: concepts/events.md
           - Automations: concepts/automations.md
-          - Filesystems: concepts/filesystems.md
-          - Block and Agent-Based Deployments:
-                - Deployments: concepts/deployments-block-based.md
-                - Infrastructure: concepts/infrastructure.md
-                - Storage: concepts/storage.md
-                - Agents: concepts/agents.md
```

Comment:
@cicdw @discdiver I think it's important to preserve access to these concepts. What do y'all think about having a _note_ or _callout_ on both the deployments and workers page that points to these docs? This way they are not featured prominently in the nav, but still very visible.

Reply:
Agree we want to make sure users can find them. We added them back to the nav in #10528 @EmilRex.